cluster/manifests/metrics-server/metrics-server-vpa.yaml

```diff
@@ -10,4 +10,9 @@ spec:
       application: metrics-server
   updatePolicy:
     updateMode: "Auto"
+  resourcePolicy:
+    containerPolicies:
+    - containerName: metrics-server
+      maxAllowed:
+        memory: {{if index .ConfigItems "prometheus_mem"}}{{.ConfigItems.prometheus_mem}}{{else}}4000Mi{{end}}
```

Comment:
Is the default for the metrics-server also 4000Mi?

Reply:
it would be better to replace it with "{{.ConfigItems.metrics_service_mem}}" and then just using else whatever it's preferred.